.travis.yml

```diff
@@ -26,4 +26,4 @@ script:
 # When branch is `master` we run `haskell-stylish` and fail if git working directory becomes dirty
 - if [ "$TRAVIS_BRANCH" == "master" ]; then make stylish && git diff-index --quiet HEAD; fi
 
-- make test
+- make test-libraries
```

Comment:
Can we put your handler test suite in here using some kind of custom stack test command?

Reply:
Like a specific`make test-sample-app-handler`?

Reply:
yeah, since we would want to run this particular one in the travis jobs